pages/apim/3.x/installation-guide/configuration/repositories/installation-guide-repositories-mongodb.adoc

```diff
@@ -53,31 +53,63 @@ management:
+    wtimeout:                   # (default is 0)
+    journal:                    # (default is true)
+
+## Socket settings
+    connectTimeOut:             # mongodb connection time out (default 1000)
+    socketTimeout:              # mongodb socket time out (default 1000)
+
+## Cluster settings
+    serverSelectionTimeout:     # mongodb server selection time out (default 1000)
+    localThreshold:             # mongodb local threshold (default 15)
+
+## Connection pool settings
     maxWaitTime:                # mongodb max wait time (default 120000)
-    socketTimeout:              # mongodb socket time out (default 0 -> never)
-    socketKeepAlive:            # mongodb socket keep alive (default false)
-    maxConnectionLifeTime:      # mongodb max connection life time (default null)
-    maxConnectionIdleTime:      # mongodb max connection idle time (default null)
-    minHeartbeatFrequency:      # mongodb min heartbeat frequency (default null)
-    description:                # mongodb description (default null)
-    heartbeatConnectTimeout:    # mongodb heartbeat connection time out (default null)
-    heartbeatFrequency:         # mongodb heartbeat frequency (default null)
-    heartbeatsocketTimeout:     # mongodb heartbeat socket time out (default null)
-    localThreshold:             # mongodb local threshold (default null)
-    minConnectionsPerHost:      # mongodb min connections per host (default null)
+    maxConnectionLifeTime:      # mongodb max connection life time (default 0)
+    maxConnectionIdleTime:      # mongodb max connection idle time (default 0)
+    connectionsPerHost:         # mongodb max connections per host (default 100)
+    minConnectionsPerHost:      # mongodb min connections per host (default 0)
+
+## Server settings
+    heartbeatFrequency:         # mongodb heartbeat frequency (default 10000)
+    minHeartbeatFrequency:      # mongodb min heartbeat frequency (default 500)
+
+## SSL settings
     sslEnabled:                 # mongodb ssl mode (default false)
-    keystore:                   # path to KeyStore (when sslEnabled is true)
-    keystorePassword:           # KeyStore password
-    keyPassword:                # password for recovering keys in the KeyStore
-    threadsAllowedToBlockForConnectionMultiplier: # mongodb threads allowed to block for connection multiplier (default null)
-    cursorFinalizerEnabled:     # mongodb cursor finalizer enabled (default false)
+    keystore:                   # path to KeyStore (when sslEnabled is true, default null)
+    keystorePassword:           # KeyStore password (when sslEnabled is true, default null)
+    keyPassword:                # password for recovering keys in the KeyStore (when sslEnabled is true, default null)
 ----
 
+[WARNING]
+====
+From version 3.10.0, Gravitee.io APIM uses the 4.1.2 version of the Java Driver. As a consequence, some settings are not available anymore.
```

Comment:
Let's please change: "As a consequence, some settings are not available anymore."

To: Therefore, some settings are no longer available.